services/web/server/src/simcore_service_webserver/projects/db.py

```diff
@@ -365,8 +367,14 @@ async def list_projects(
                     | (users.c.name.ilike(f"%{search}%"))
                 )
 
-            # Default ordering
-            query = query.order_by(desc(projects.c.last_change_date), projects.c.id)
+            if order_by:
+                if order_by.direction == OrderDirection.ASC:
+                    query = query.order_by(sa.asc(order_by.field))
+                else:
+                    query = query.order_by(sa.desc(order_by.field))
+            else:
```

Comment:
if you have your default set in the openapi, then you do not need that else case

Reply:
check first that you are not calling this function from another place other than the handlers. In that case, the default belongs here and not in the rest API

Reply:
I have added default in the openapi specs and also add default here in this function.